src/main/java/uk/gov/ch/util/ApiLogger.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.ch.util;
```

Comment:
Having this new class is a divergence from how OracleQueryAPI currently logs - it seems to just use the structured logging classes directly.

I think this class was added to the ROE API as a convenience but does it really need to be introduced here too? Oracle Query API is presumably logging just fine with the current approach?

Reply:
I believe we need to use structured logging and the ROE API way of doing that seemed the right approach.